helix-view/src/view.rs

```diff
@@ -213,9 +207,10 @@ impl View {
         doc: &Document,
         scrolloff: usize,
     ) -> Option<ViewPosition> {
+        let view_offset = doc.get_view_data(self.id)?.view_position;
```

Comment:
thee still seem to be a lot of places that access `view_position` directly instead of using the `view_offset` function. I would like to avoid that unless its absoluetly necssary (which I don't think its anywhere as far as I can tell).

Reply:
Done (with one exception that I will highlight), and I made `view_data.view_position`, `Document::view_data` and `Document::view_data_mut` private